ui/src/main/java/org/pentaho/di/ui/trans/steps/transexecutor/TransExecutorDialog.java

```diff
@@ -684,7 +684,7 @@ protected void getParametersFromTrans( TransMeta inputTransMeta ) {
 
         TableItem item = new TableItem( wTransExecutorParameters.table, SWT.NONE );
         item.setText( 1, Const.NVL( name, "" ) );
-        item.setText( 3, Const.NVL( desc, "" ) );
+        item.setText( 3, Const.NVL( inputTransMeta.getParameterDefault( name ), "" ) );
```

Comment:
So should anything be done with the desc variable? I see that you're getting the parameter default, but it appears to me that if anything used to show up in desc, it won't now. Would it make sense to do something like:

String str = inputTransMeta.getParameterDefault( name );
str = ( str != null ? str : ( desc != null ? desc : "" ) );
item.setText( 3, Const.NVL( str ) );